src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1814,18 +1821,22 @@ namespace FanCoilUnits {
 			CoolingCoilName = FanCoil( FanCoilNum ).CCoilName;
 			CoolingCoilType = FanCoil( FanCoilNum ).CCoilType;
 		}
-		if ( ZoneSizingRunDone ) {
-			SetCoilDesFlow( CoolingCoilType, CoolingCoilName, FinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, ErrorsFound );
-			SetCoilDesFlow( FanCoil( FanCoilNum ).HCoilType, FanCoil( FanCoilNum ).HCoilName, FinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow, ErrorsFound );
-		} else {
-			SetCoilDesFlow( CoolingCoilType, CoolingCoilName, FanCoil( FanCoilNum ).MaxAirVolFlow, ErrorsFound );
-			SetCoilDesFlow( FanCoil( FanCoilNum ).HCoilType, FanCoil( FanCoilNum ).HCoilName, FanCoil( FanCoilNum ).MaxAirVolFlow, ErrorsFound );
-		}
+//		if ( ZoneSizingRunDone ) {
+//			SetCoilDesFlow( CoolingCoilType, CoolingCoilName, FinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, ErrorsFound );
+//			SetCoilDesFlow( FanCoil( FanCoilNum ).HCoilType, FanCoil( FanCoilNum ).HCoilName, FinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow, ErrorsFound );
+//		} else {
+//			SetCoilDesFlow( CoolingCoilType, CoolingCoilName, FanCoil( FanCoilNum ).MaxAirVolFlow, ErrorsFound );
+//			SetCoilDesFlow( FanCoil( FanCoilNum ).HCoilType, FanCoil( FanCoilNum ).HCoilName, FanCoil( FanCoilNum ).MaxAirVolFlow, ErrorsFound );
+//		}
```

Comment:
@mjwitte I had no idea that parents were setting child air flow rate this way. And it's wrong to boot. This is a fan coil that has a single operating air flow rate for both cooling and heating. The FinalZoneSizing flows were passed to the coil if it was a sizing run and these are different for cooling and heating. Also, if this is done, an autosized coil now appears to be user specified sizing when the coil calls RequestSizing instead of design size. This might take some time to sort out.